src/gwt/ant

```diff
@@ -15,6 +15,7 @@ if [ -z "${JAVA_HOME}" ]; then
 	read -r -d '' homes <<- EOF
 	/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home
 	/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
+	/opt/homebrew/opt/openjdk@11
```

Comment:
Did you mean for this change to be included in the PR?

Reply:
I did; this is a developer-only change not worth a separate PR.